src/main/webapp/search-interview.html

```diff
@@ -44,86 +44,18 @@ <h5 class="text-center lead">Search for possible times in which you will be inte
       <form>
         <div class="form-group">
           <!-- TODO: add parameters for this form. -->
```

Comment:
Drop this todo?

Reply:
No, this is some of the qualifications and shadow work.